docs/FAQ.md

```diff
@@ -63,8 +63,9 @@ easy way to see the evolution of the commit's contents.
 
 ### Can I prevent Jujutsu from recording my unfinished work? I'm not ready to commit it.
 
-Jujutsu automatically records new files in the current working-copy commit and
-doesn't provide a way to prevent that.
+You can set `snapshot.auto-track` to only start tracking new files matching the
```

Comment:
I think this passage might make more sense for the question about scratch files below.

For *this* question, you could change the paragraph to something like:

> Jujutsu automatically records changes to tracked files in the current working copy commit...

I think we could not mention "new" files at all.

Cc #4320, which might need to be adjusted once this PR goes in.

Reply:
I moved it to that section.

(But the current text here ("Jujutsu automatically records new files in the current working-copy commit and
doesn't provide a way to prevent that.") does sound very much related to this feature too, doesn't it? I'm not sure I understand the difference between the two entries.)